packages/@react-native-windows/cli/src/commands/initWindows/initWindows.ts

```diff
@@ -125,11 +125,28 @@ export class InitWindows {
     return name;
   }
 
+  protected printTemplateList() {
+    if (this.templates.size === 0) {
+      console.log('\nNo templates found.\n');
+      return;
+    }
+
+    for (const [key, value] of this.templates.entries()) {
+      console.log(`\n${key} - ${value.name}\n    ${value.description}`);
```

Comment:
Can you somehow indicate the default template? There should be a boolean property on the template you can use.

Reply:
Yes! I got it:

![image](https://github.com/user-attachments/assets/a1b77972-b83b-4051-834e-ee60b30fafaf)